### PR TITLE
Cleanup of pipeline, added automatic deploy to preint

### PR DESCRIPTION
**Pull request by brunoOSI**
Cleanup of Jenkinsfile structure

### Jira Story

- [Name of Story INT-NNN](https://osi-cwds.atlassian.net/browse/INT-NNN)

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -54,61 +50,65 @@ node('intake-slave') {
                 }
             }
         }
-    }
-    catch (e) {
+
+        // Generate reports
```
Comment (edgibbs) [inline Jenkinsfile]: remove comment

File: Jenkinsfile [hunk 2]
```diff
@@ -54,61 +50,65 @@ node('intake-slave') {
...
-              slackMessage = "${pipelineStatus}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' completed for branch '${branch}' (${env.BUILD_URL})"
+            slackSend channel: "#tech-intake", baseUrl: 'https://hooks.slack.com/services/', tokenCredentialId: 'slackmessagetpt2', color: slackAlertColor, message: slackMessage 
 
-              if(pipelineStatus == 'FAILED') {
-                slackAlertColor = failureColor
-                slackMessage = "${pipelineStatus}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' in stage '${curStage}' for branch '${branch}' (${env.BUILD_URL})"
-              }
-
-              slackSend channel: "#tech-intake", baseUrl: 'https://hooks.slack.com/services/', tokenCredentialId: 'slackmessagetpt2', color: slackAlertColor, message: slackMessage
+            // Trigger a deploy on preint
```
Comment (edgibbs) [inline Jenkinsfile]: remove comment

File: Jenkinsfile [hunk 3]
```diff
@@ -54,61 +50,65 @@ node('intake-slave') {
...
+              sh "curl -v http://${JENKINS_USER}:${JENKINS_API_TOKEN}@jenkins.mgmt.cwds.io:8080/job/preint/job/deploy-intake/buildWithParameters?token=${JENKINS_TRIGGER_TOKEN}&cause=Caused%20by%20Build%20${env.BUILD_ID}"
+              pipelineStatus = 'SUCCEEDED'
+              currentBuild.result = 'SUCCESS'
+            }
           }
-          catch(e) { /* Nothing to do */ }
-        }
 
-        stage('Clean') {
+          // CLEAN
```
Comment (edgibbs) [inline Jenkinsfile]: remove and why all caps

**Comment by brunoOSI** [comment]
retest this please

**Pull request merged**